test/tests/php-fpm-hello-web/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+# Build a client image with cgi-fcgi for testing
+client_image="php-fpm-client-$RANDOM-$RANDOM"
+docker build -q -t "$client_image" - > /dev/null <<EOF
```

Comment:
Hmmm, so an image with all the cli's straight from debian packages to test all the others? (`mysql-client`, `libfcgi0ldbl`, etc?)